### PR TITLE
Add CityDAO protocol info

### PR DESCRIPTION
**Pull request by cryptosabago**

### CODE REVIEW

**Comment by cryptosabago** [comment]
@Rashmi-278 do you have any idea how i can fix the `yarn run` error? Not sure if it is related to my recent OS upgrade...

**Comment by CarlGocht** [comment]
> @Rashmi-278 do you have any idea how i can fix the `yarn run` error? Not sure if it is related to my recent OS upgrade...

Sorry about that I think it was my fault. Can you rebase on main and try again

File: protocols/cityDAO/index.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "cname": "CityDAO",
```
Comment (Rashmi-278) [inline protocols/cityDAO/index.json:2]: All small case please :)

File: protocols/cityDAO/index.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "description": "CityDAO is building the city of the future on the ethereum blockchain by tokenizing land, rights, and governance.",
+  "path": "CityDAO",
```
Comment (Rashmi-278) [inline protocols/cityDAO/index.json:4]: All small case :)

File: protocols/cityDAO/index.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "type": "snapshot",
+  "isEnabled": true,
+  "discourseForum": {
+    "url": "https://forum.citydao.io/c/cips/",
```
Comment (Rashmi-278) [inline protocols/cityDAO/index.json:8]: Please remove /c/cips/

Comment (Rashmi-278) [reply protocols/cityDAO/index.json:8]: Just https://forum.citydao.io/

**Review by Rashmi-278 — CHANGES_REQUESTED** [review]

**Review by Rashmi-278 — APPROVED** [review]

**Pull request merged**